src/input_xml.F90

```diff
@@ -164,12 +165,28 @@ subroutine read_settings_xml()
       end if
     end if
 
+    ! Find the windowed multipole library
+    if (run_mode /= MODE_PLOTTING) then
+      if (.not. check_for_node(doc, "multipole_library") .and. &
+           run_mode /= MODE_PLOTTING) then
+        ! No library location specified in settings.xml, check
+        ! environment variable
+        call get_environment_variable("OPENMC_MULTIPOLE_LIBRARY", env_variable)
+        path_multipole = trim(env_variable)
```

Comment:
Good point, but I decided to put this check in `ace % read_ace_xs` where we can also catch the case of a user specifying the environment variable, but specifying it incorrectly.